ete3/treeview/faces.py

```diff
@@ -2420,3 +2420,68 @@ def _width(self):
     def _height(self):
         return self.height
 
+
+class ArrowFace(Face)
```

Comment:
you are missing the ':'
(fix locally in your branch, then commit, then push, and the pull request will get updated automatically).